src/providers/MeetingEventProvider/index.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { createContext, useState, useContext, useEffect } from 'react';
+import { EventName, EventAttributes } from 'amazon-chime-sdk-js';
+
+import { useMeetingManager } from '../MeetingProvider';
+
+export type MeetingEventProviderContextType = {
+  name: EventName;
+  attributes: EventAttributes;
+} | undefined;
+
+export const MeetingEventProviderContext = createContext<MeetingEventProviderContextType>(undefined);
+
+const MeetingEventProvider: React.FC = ({ children }) => {
+  const [eventData, setEventData] = useState<MeetingEventProviderContextType>();
+  const meetingManager = useMeetingManager();
+
+  useEffect(() => {
+    function eventDataUpdateCb(name: EventName, attributes: EventAttributes) {
+      setEventData({name, attributes});
+    }
+
+    meetingManager.subscribeToEventDidReceive(eventDataUpdateCb);
+
+    return () => {
+      meetingManager.unsubscribeFromEventDidReceive(eventDataUpdateCb);
+    };
+  }, []);
+
+  return (
+    <MeetingEventProviderContext.Provider value={eventData}>
+      {children}
+    </MeetingEventProviderContext.Provider>
+  );
+};
+
+const useMeetingEventData = (): MeetingEventProviderContextType => {
```

Comment:
How about just `useMeetingEvent` or `useMeetingEvents` instead of introducing a new term `EventData`?

Reply:
Yes, I like it, changed the `useMeetingEventData` -> `useMeetingEvent`. Updated the references elsewhere too.